Source/WebCore/html/HTMLElement.cpp

```diff
@@ -776,6 +776,28 @@ void HTMLElement::setTranslate(bool enable)
     setAttributeWithoutSynchronization(translateAttr, enable ? "yes"_s : "no"_s);
 }
 
+bool HTMLElement::cue() const
+{
+    const AtomString& value = attributeWithoutSynchronization(cueAttr);
+    return equalLettersIgnoringASCIICase(value, "true"_s) ? true : false;
```

Comment:
Sorry for not noticing this sooner, but why are we not making these boolean attributes? True if present, false if absent.

Reply:
As in, just return `hasAttributeWithoutSynchronization()` here and either add with the empty string or remove below.

Reply:
Oh yeah, right! oops. I'll a test for this also.